packages/core/src/components/common/PreviewTag/PreviewTag.tsx

```diff
@@ -0,0 +1,58 @@
+import { useRouter } from 'next/router'
+import { useEffect } from 'react'
+import { Tag as UITag } from '@faststore/ui'
+import Section from 'src/components/sections/Section/Section'
+
+import styles from './section.module.scss'
+
+export interface PreviewTagProps {
+  text?: string
+  exitUrl?: string
+}
+
+function PreviewTag({ text = 'Preview', exitUrl }: PreviewTagProps) {
+  const router = useRouter()
```

Comment:
todo: remove useRouter mention, I believe useRouter is not being used here anymore
```suggestion
import { useEffect } from 'react'
import { Tag as UITag } from '@faststore/ui'
import Section from 'src/components/sections/Section/Section'

import styles from './section.module.scss'

export interface PreviewTagProps {
  text?: string
  exitUrl?: string
}

function PreviewTag({ text = 'Preview', exitUrl }: PreviewTagProps) {
```